src/components/layouts/Notification.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useAuth } from '../../context/AuthProvider';
+import Image from 'next/image';
+import icBellAlert from '/assets/icons/ic_bell_alert.svg';
+import icBellEmpty from '/assets/icons/ic_bell_empty.svg';
+import styles from './Notification.module.css';
+
+const Notification = () => {
+  const [notifications, setNotifications] = useState([]);
+  const [showNotifications, setShowNotifications] = useState(false);
+  const { user } = useAuth();
+
+  useEffect(() => {
+    if (user) {
+      fetchNotifications();
+    }
+  }, [user]);
+
+  const fetchNotifications = async () => {
+    try {
+      const response = await axios.get(
+        `/api/notifications/users/${user.id}/notifications`
+      );
+      setNotifications(response.data);
+    } catch (error) {
+      console.error('알림을 가져오는 중 오류가 발생했습니다:', error);
+    }
+  };
+
+  const markAsRead = async (notificationId) => {
+    try {
+      await axios.put(
+        `/api/notifications/notifications/${notificationId}/read`
+      );
+      fetchNotifications(); // 알림 목록 갱신
+    } catch (error) {
+      console.error('알림 읽음 처리 중 오류가 발생했습니다:', error);
+    }
+  };
```

Comment:
```suggestion
// service/api/notifications.js
import axios from 'axios';

export const fetchNotifications = async (userId) => {
  return await axios.get(`/api/notifications/users/${userId}/notifications`);
};

export const markAsRead = async (notificationId) => {
  return await axios.put(`/api/notifications/notifications/${notificationId}/read`);
};
```

현재 API 호출이 페이지에 직접 작성되어 있습니다. 이를 `service` 폴더 내에 새로운 파일로 분리하여 API 호출 함수를 정의하고, 페이지에서 해당 함수를 가져와 사용해주세요

Reply:
넵  알겠습니다!